qfieldsync/core/cloud_api.py

```diff
@@ -252,22 +375,33 @@ def server_url(self):
 
         return re.sub(r"([^:]/)(/)+", r"\1", url)
 
+    def get_auth_capabilities(self) -> QNetworkReply:
+        """Get authentication capabilities from a server"""
+        reply = self.cloud_get("auth/providers/")
+        return reply
+
     def auto_login_attempt(self):
```

Comment:
```suggestion
    def auto_login_attempt(self) -> None:
```
Feel free to ignore if you don't have other work on this PR